.gitignore

```diff
@@ -1 +1,2 @@
-
+text.nd
+node_modules
```

Comment:
- [ ] Great job following the requirements of the project. 🙌😊😃 However, I can see that there is a little typo at line 1 which was actually **text.md**, but is written as **text.nd**, due to which, it is not ignored by the git. So kindly fix this typo to meet the project requirements. Also, in the requirements, the name of the file is mentioned as **test.md**, so please always make sure to keep a sharp eye on the requirements, as doing so is always best to deliver a complete project. 😃